src/GovUk.Education.ExploreEducationStatistics.Common/Services/Interfaces/ITableStorageService.cs

```diff
@@ -0,0 +1,10 @@
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos.Table;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces
+{
+    public interface ITableStorageService
+    {
```

Comment:
Quick question - why do we need to create tables on the fly? Or am I misunderstanding things? Or is it for convenience?

Reply:
Convince, the table might not exist in an environment so this will create it the first time it it doesn't exist.